APCAppCore/APCAppCore/Library/APCTaskImporter.h

```diff
@@ -49,7 +49,7 @@
  
  See the .m file for technical notes on this method.
  */
-- (void) processTasks: (NSArray *) arrayOfTasks
+- (BOOL) processTasks: (NSArray *) arrayOfTasks
```

Comment:
What's the reason for changing this method's signature? Doesn't *seem* like something the analyzer would advise, and I don't see any callers that have been updated in this PR.